static/js/topic_page/block.tsx

```diff
@@ -67,6 +67,10 @@ export function Block(props: BlockPropType): JSX.Element {
 function renderTiles(tiles: Tile[], props: BlockPropType): JSX.Element {
   const tilesJsx = tiles.map((tile) => {
     const id = randDomId();
+    const enclosedPlaceType =
+      tile.containedPlaceTypes && tile.containedPlaceTypes[props.place.types[0]]
```

Comment:
might want to check that props.place.types isn't an empty array before trying to access it. And same in the MainPane component

Reply:
done - added fallback code in both places, and especially in topic_page.ts so we have a consistent fallback